recipes/msgpack/all/conanfile.py

```diff
@@ -14,6 +14,8 @@ class MsgpackConan(ConanFile):
     settings = "os", "arch", "build_type", "compiler"
     options = {"fPIC": [True, False], "shared": [True, False], "header_only": [True, False], "with_boost": [True, False]}
     default_options = {"fPIC": True, "shared": False, "header_only": False, "with_boost": False}
+    _autotools = None
```

Comment:
This line can be removed.

Reply:
You wrote above

>  add a _autotools=None statement at the top

Did I misunderstand something ? I have to admit I added this without understanding why :/

Reply:
The `_cmake=None` line is required. Don't remove that one.
You don't use the `_autotools`, so it can go away.

This line is required because otherwise, the line `if self._cmake:` in `_configure_cmake` would throw an `AttributeError`. (try it 😄 )

Reply:
Oh yes I know that, I guess you mean `_cmake=None` instead of `_autotools=None`, that confused me. I'll remove the extra `_autotools=None`